src/cairoUtilFuncGen/base.ts

```diff
@@ -117,6 +112,7 @@ export function delegateBasedOnType<T>(
   valueFunc: (type: TypeNode) => T,
 ): T {
   if (type instanceof PointerType) {
+    assert(false);
```

Comment:
Either remove the assertion or the whole if its a dead code

Reply:
Using assert so I could get a trace when debugging. Leaving it as it was.